.github/workflows/publish_to_pypi_ci.yml

```diff
@@ -37,16 +37,16 @@ jobs:
                   name: python-package-distributions
                   path: ./wheelhouse/*.whl
 
-
-    publish-to-testpypi:
-      name: Publish Python distribution to TestPyPI
+          
+    publish-to-pypi:
+      name: Publish Python distribution to PyPI
       needs:
       - build
       runs-on: ubuntu-latest
 
       environment:
-        name: testpypi
-        url: https://test.pypi.org/p/qdd
+        name: pypi
+        url: https://pypi.org/p/qdd
```

Comment:
Is that an expected behavior that https://pypi.org/p/qdd returns 404 not found error?  Also "QDD" we are currently developing didn't come up when I searched for that in PyPI.

Reply:
As the token that @kkyusuke created has not been used yet, it remains to be a "pending" publisher.
According to the docs, the project does not exist for a pending publisher.
https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/

Reply:
Once the distribution is published to PyPI, the project page will be created with README and wheel files.